packages/demos/src/server-side-rendering/integrator.tsx

```diff
@@ -19,6 +18,18 @@ function getSerializedStatesFromDom(): string | undefined {
   return (scriptElement && scriptElement.textContent) || undefined;
 }
 
+function getUrlsForHydrationFromDom(): string[] | undefined {
```

Comment:
Please always return an empty array

Reply:
and then check the length in the guard below? why?

Reply:
the guard is only theoretical anyway, so maybe getting rid of it and returning an empty array here is indeed better

Reply:
fixed (sorry, I accidentally amended)